psi4/CMakeLists.txt

```diff
@@ -192,6 +192,13 @@ configure_file(run_psi4.py psi4 @ONLY)
 install(PROGRAMS ${CMAKE_BINARY_DIR}/psi4
         DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})
 
+if(WIN32)
```

Comment:
I might have missed it, but it WIN64 required here as well?

Reply:
There is no `WIN64`: https://cmake.org/cmake/help/latest/variable/WIN32.html